sync_ci/pkg/parser/ci_log.go

```diff
@@ -30,6 +30,10 @@ func ParseCILog(job string, ID int64) (map[string][]string, error) {
 	}
 	res := map[string][]string{
 	}
+
+	// to support regexpRules, store all content in this slice.
+	regexRulesLines := lines
```

Comment:
worry about the memory.

Reply:
me too. Maybe we could remove same ones in key value pairs to avoid large memory usage.